org.eclipse.wb.swing/src/org/eclipse/wb/internal/swing/preferences/colorchooser/IColorChooserPreferenceConstants.java

```diff
@@ -0,0 +1,20 @@
+package org.eclipse.wb.internal.swing.preferences.colorchooser;

```

Comment:
Copyright must be added to all new files!

Reply:
I understand I will do so. Thank you. :)